src/test/java/domain/JanggiTest.java

```diff
@@ -0,0 +1,66 @@
+package domain;
+
+import domain.position.Position;
+import domain.position.Route;
+import java.util.List;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class JanggiTest {
+
+    @Test
+    @DisplayName("졸의 이동경로를 구한다.")
+    void test1() {
+        // given
+        Janggi janggi = new Janggi();
+
+        // when
+
+        List<Route> routes = janggi.searchAvailableRoutes(new Position(0, 3));
```

Comment:
불필요한 줄바꿈인 것 같아요. 🦅👁️‍🗨️ 

```suggestion
        List<Route> routes = janggi.searchAvailableRoutes(new Position(0, 3));
```